src/main/java/kostka/moviecatalog/controller/AdministrationController.java

```diff
@@ -100,8 +100,6 @@ public String createMovie(final @Valid @ModelAttribute MovieFormDto dto,
             addModelAttributes(model, "Movie cannot be created.");
             return ADMIN_VIEW;
         }
-
-        redirectAttributes.addFlashAttribute(SUCCESS, "Movie is successfully created.");
```

Comment:
Why did you get rid of this?
Flash messagse are the coolest thing from old web :D